### PR TITLE
Editorial: Improved introduction on deprecation

### PR DESCRIPTION
**Pull request by tfrauenstein**

### CODE REVIEW

File: chapters/deprecation.adoc [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
 
-Sometimes it is necessary to phase out an API endpoint (or version).
-I.e. this may be necessary if a field is no longer supported in the
-result or a whole business functionality behind an endpoint has to be
-shut down. There are many other reasons as well.
+Sometimes it is necessary to phase out an API endpoint (or version), for instance, 
+if a field is no longer supported in the result or a whole business functionality 
+behind an endpoint has to be shut down. There are many other reasons as well. 
+Of course, these changes are breaking changes as long as these endpoints are still 
+used by consumers and are not allowed. Deprecation rules of play have to be applied,
```
Comment (tkrop) [inline chapters/deprecation.adoc]: How about rewriting: `Of course, these changes are breaking changes as long as these endpoints are still used by consumers and are not allowed.` to `Of course, these breaking changes are forbidden as long as these endpoints are still used by consumers.`

Starting a new paragraph after this sentence makes sense in my opinion. Besides, can you split up the next complex sentence into more simpler sentences?

Comment (tfrauenstein) [reply chapters/deprecation.adoc]: - not exactly: it are only breaking changes, if used by consumers -- I changed the sentence a bit.
- I would avoid new paragraph with just one sentence; splitting feels not necessary here.

File: chapters/deprecation.adoc [hunk 2]
```diff
@@ -4,10 +4,10 @@
 Sometimes it is necessary to phase out an API endpoint (or version), for instance, 
 if a field is no longer supported in the result or a whole business functionality 
 behind an endpoint has to be shut down. There are many other reasons as well. 
-Of course, these changes are breaking changes as long as these endpoints are still 
-used by consumers and are not allowed. Deprecation rules of play have to be applied, 
-as to make sure that alignment about imposed consumer side changes has taken place 
-and deprecated endpoints are not used my actively maintained consumers before 
+Of course, as long as these endpoints are still used by consumers these are breaking
```
Comment (tkrop) [inline chapters/deprecation.adoc]: I would remove `Of course` here, because not going well with the `as long as`.

Comment (tfrauenstein) [reply chapters/deprecation.adoc]: done

Comment (tkrop) [reply chapters/deprecation.adoc]: I would still favor my suggestion `(Of course, )these breaking changes are forbidden as long as these endpoints are still used by consumers.` as it reads without the complex structure having partial sentence connected by `and`.

File: chapters/deprecation.adoc [hunk 3]
```diff
@@ -1,10 +1,14 @@
...
-result or a whole business functionality behind an endpoint has to be
-shut down. There are many other reasons as well.
+Sometimes it is necessary to phase out an API endpoint (or version), for instance, 
+if a field is no longer supported in the result or a whole business functionality 
+behind an endpoint has to be shut down. There are many other reasons as well. 
+As long as these endpoints are still used by consumers these are breaking 
+changes and not allowed. Deprecation rules of play have to be applied, 
+as to make sure that alignment about imposed consumer changes has taken place 
+and deprecated endpoints are not used by actively maintained consumers before 
+API changes become effective.
```
Comment (tkrop) [inline chapters/deprecation.adoc]: This sentence should really be split up and simplified. I'm used to such sentences as I write them myself, but even I have problems to get the intended meanings.

**Comment by tkrop** [comment]
:+1:

**Comment by codekultur** [comment]
👍

**Pull request merged**